modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CP2TeamMembersListing.testcase

```diff
@@ -0,0 +1,230 @@
+			CustomerPortal.deleteEntryAccountFlag(entry = "ERC-002");
+		}
+	}
+
+	@description = "Automation ID: LPS-149756 | Test Summary: Verify if the '+ Invite' button is visible for administrator roles only"
+	@priority = "5"
+	test CanViewTheInviteButton {
+		property portal.upstream = "quarantine";
+
+		// "Reason: Not eble to reproduce scenario without OKTA workaround."
+
+		task ("Go to CP2.0") {
+			ApplicationsMenu.gotoSite(key_site = "Customer Portal Site");
+		}
+
+		task ("Login in Okta") {
+			CustomerPortal.loginOktaToUpdateCP();
+
+			SelectWindow(locator1 = "title=Home - Customer Portal Site - Liferay DXP");
+
+			Refresh();
+		}
+
+		task ("Get into the project and go to Team Members page") {
+			CustomerPortal.enterOnProject(title = "Test Account DXP");
+
+			CP2NavigationMenu.goToTeamMembers();
+		}
+
+		task ("Assert that the button is not on the screen") {
+			AssertElementNotPresent(locator1 = "CP2TeamMembers#INVITE_BUTTON");
+		}
+
+		task ("Change the user role in the project") {
+			Accounts.openAccountsAdmin();
+
+			CustomerPortal.accountTableProject(title = "Test Account DXP");
+
+			Accounts.addAssigneeToRoleViaUsersTab(
+				accountRoleTitle = "Account Administrator",
+				accountUserName = "Test Test");
+		}
+
+		task ("Get into the project and go to Team Members page") {
+			ApplicationsMenu.gotoSite(key_site = "Customer Portal Site");
+
+			CP2TeamMembers.teamMembersFlow();
+		}
+
+		task ("Assert that the Invite button is present on the screen") {
+			AssertElementPresent(locator1 = "CP2TeamMembers#INVITE_BUTTON");
+		}
+
+		task ("Delete the entries and objects created") {
+			CustomerPortal.deleteEntryAccountFlag(entry = "ERC-002");
+		}
+	}
+
+	@description = "Automation ID: LPS-149748 | Test Summary: Verify if in the third column next to the 'Support Seat' heading the information icon has a tooltip"
+	@priority = "4"
+	test CanViewTooltipInSupportSeat {
+		property portal.upstream = "quarantine";
+
+		// "Reason: Not eble to reproduce scenario without OKTA workaround."
+
+		task ("Change the user role in the project") {
+			Accounts.openAccountsAdmin();
+
+			CustomerPortal.accountTableProject(title = "Test Account DXP");
+
+			Accounts.addAssigneeToRoleViaUsersTab(
+				accountRoleTitle = "Account Administrator",
+				accountUserName = "Test Test");
+		}
+
+		task ("Go to CP2.0") {
+			ApplicationsMenu.gotoSite(key_site = "Customer Portal Site");
+		}
+
+		task ("Login in Okta") {
+			CustomerPortal.loginOktaToUpdateCP();
+
+			SelectWindow(locator1 = "title=Home - Customer Portal Site - Liferay DXP");
+
+			Refresh();
+		}
+
+		task ("Get into the project and go to Team Members page") {
+			CP2TeamMembers.teamMembersFlow();
+		}
+
+		task ("View the help information") {
+			Tooltip.viewHelpInfo(info = "The limit of support seats available counts the total of Administrators & Requesters roles assigned due to both have role permissions to open support tickets.");
+		}
```

Comment:
@luanakarla since we are only adding this validation to this task, do you think this assert could be added to AssertAllItemsInTheTable test? It looks like we are performing the same steps. We could combine both tests into only 1 and decrease the use of CI servers.

Reply:
Done.